builds/e2e/longhaul.yaml

```diff
@@ -13,6 +13,8 @@ variables:
   #pool.name: ''
   #container.registry: ''
   #snitch.storage.account: ''
+  #logAnalytic.enabled  
```

Comment:
why not be consistent as template parameter?
```suggestion
  #analyzer.logAnalytic.enabled  
```

Reply:
No. that would be too confusing without comment on which variable is being refer when you're setting them.

Reply:
I'm very tempted to do `kvLogAnalyticWorkspaceId` and `vsLogAnalyticLogType` to show that one is from key vault, the other is from VSTS pipeline.